src/components/Charts/MiniProgress/index.js

```diff
@@ -1,11 +1,21 @@
 import React from 'react';
 import { Tooltip } from 'antd';
+import { formatMessage } from 'umi/locale';
 
 import styles from './index.less';
 
-const MiniProgress = ({ target, color = 'rgb(19, 194, 194)', strokeWidth, percent }) => (
+const MiniProgress = ({
+  targetLabel,
+  target,
+  color = 'rgb(19, 194, 194)',
+  strokeWidth,
+  percent,
+}) => (
   <div className={styles.miniProgress}>
-    <Tooltip title={`目标值: ${target}%`}>
+    <Tooltip
+      title={`${targetLabel ||
+        formatMessage({ id: 'component.miniProgress.tooltipDefault' }).concat(': ')} ${target}%`}
```

Comment:
This code is not beautiful enough

Reply:
Could you please detail ? Are you talking about using _concat_ during the default value of the property ?